instance/models/mixins/openedx_database.py

```diff
@@ -228,6 +229,12 @@ def set_field_defaults(self):
         But since we add suffixes to mysql_user to generate unique user names
         for different services (e.g. xqueue) we don't want to use the maximum length here.
         """
+        if not self.use_ephemeral_databases:
+            if not self.mysql_server and MySQLServer.objects.exists():
+                self.mysql_server = random.choice(MySQLServer.objects.all())  # FIXME: Proper selection algorithm
```

Comment:
@itsjeyd You can take a look how I implemented this for the load balancers here: https://github.com/open-craft/opencraft/pull/145/files#diff-b5582c15d549b6740502b12f1b5fd04bR66

The manager class also contains logic that allows to configure a single load-balancer via the Django settings, which is needed for easy configuration of the integration tests.  I suggest you use the same logic here.  It should be possible to factor out the common parts.